ui/packages/catalog-ui-search/src/main/webapp/component/result-form/result-form.view.js

```diff
@@ -88,11 +88,10 @@ module.exports = Marionette.LayoutView.extend({
     )
   },
   setupTitleInput: function() {
-    let currentValue = this.model.get('name') ? this.model.get('name') : ''
     this.basicTitle.show(
       new PropertyView({
         model: new Property({
-          value: [currentValue],
+          value: [this.model.get('title')],
```

Comment:
Is this guaranteed to be here? You might want to just update this deleted line to use `title` instead.
```java
    let currentValue = this.model.get('title') ? this.model.get('title') : ''
```

Reply:
Every form gets the default defined in `search-form.js`, which I changed to `''`, so title is guaranteed to be at least the empty string